config/locales/en.yml

```diff
@@ -2551,3 +2554,9 @@ en:
       comment_hint: "Don't include any personal or sensitive information."
       send: Send
       rating: 'Overall, how satisfied were you with this service?'
+
+
+
+  # global
+  global_no: "No"
```

Comment:
i'd put these at the top under `en:`
you could also do this as further refactoring to anchor and reuse

```
en: 
  # anchor
  global_yes: &global_yes Yes
  global_no: &global_no No

  # and use
  my_view:
     anwser_yes: *global_yes
     anwser_no: *global_no
```
we should also probably have a standard/convention relating to use of quotes (single quotes are better IMO and are the default if no quotes specified, so no quotes actually needed except for interpolation and escaping chars)

Reply:
👍